### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
@EdwardBetts please review

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -2556,6 +2563,7 @@ confids->confides
 configire->configure
 configration->configuration
 configrations->configurations
+configruated->configuration
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: configured

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -4281,6 +4293,7 @@ eigth->eighth, eight,
 eihter->either
 einstance->instance
 eiter->either
+either either->either
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: I thought that we were unable to find such errors. Does this actually work? If not, we should add a check in the dict checking tests that there is no whitespace in the dict keys

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: My experience with spellintian is that it's a bit of a nightmare of false positives anyway. It should probably just be added as another option which does duplicate word checking, but spellintian does it across line boundaries so it catches things like fi twice in bash scripts, but I suspect line boundaries are the most likely place for people to make duplicate word entries, so it's lose lose (which ironically is an intentional duplication).

**Comment by larsoner** [comment]
We should merge #616 before this one

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: sorry, so should I remove this? BTW.. will codespell still catch typos like this ?

Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: No it won't in its current form. Eventually it might be worth adding such two-word combos but someone would have to volunteer and be motivated to do it.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Well, said person should then remember to re-implement 
https://github.com/codespell-project/codespell/pull/616/files#diff-3bddc99f1f6dffc98ea11be5a5a31ae9

Comment (larsoner) [reply codespell_lib/data/dictionary.txt]: Indeed, as noted in the description of that PR

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Although I'd say yes to the related words, but not the duplicates stripped out of that. However I imagine the default, and custom regexes, are looking for words, so it's more a case of comparing the current and following word and then probably only suggesting not auto-replacing. E.g. my regex matches underscore separated variable names.

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: P.S. This still needs removing to fix the test @luzpaz .

**Comment by luzpaz** [comment]
Ready for merge
